src/backends/plotly.jl

```diff
@@ -324,10 +325,21 @@ function plotly_layout(plt::Plot)
 
         # if any(is3d, seriesargs)
         if is3d(sp)
+            azim = sp[:camera][1] - 90 #convert azimuthal to match GR behaviour
+            theta = 90 - sp[:camera][2] #spherical coordinate angle from z axis
             d_out[:scene] = KW(
                 Symbol("xaxis$spidx") => plotly_axis(sp[:xaxis], sp),
                 Symbol("yaxis$spidx") => plotly_axis(sp[:yaxis], sp),
                 Symbol("zaxis$spidx") => plotly_axis(sp[:zaxis], sp),
+
+                #2.6 multiplier set camera eye such that whole plot can be seen
```

Comment:
Is 2.6 independent of the plots' `size`?

Reply:
Oops `size` does mess with the view a bit for plotly not sure how to fix this currently.

Reply:
So I played around with the window `size` a bit more and the 2.6 works as long as the window isn't so small that it can no longer fit the plot.